app-frontend/src/app/pages/market/model/model.html

```diff
@@ -0,0 +1,179 @@
+        </div>
+        <div class="model-detail-section">
+          <div class="model-detail-header">
+            <h5>Tags</h5>
+          </div>
+          <div class="sidebar-category">
+            <button class="btn btn-tag"
+                    ng-repeat="tag in $ctrl.modelData.tags">
+              {{tag}}
+            </button>
+          </div>
+        </div>
+        <div class="model-detail-section">
+          <div class="model-detail-header">
+            <h5>Categories</h5>
+          </div>
+          <div class="sidebar-category">
+            <button class="btn btn-tag"
+                    ng-repeat="category in $ctrl.modelData.categories">
+              {{category}}
+            </button>
+          </div>
+        </div>
+        <div class="model-detail-section">
+          <div class="model-detail-header">
+            <h5>Usage License</h5>
+            <p>
+              No license information was provided. If this work was prepared by
+              Raster Foundry as part of the official Model marketplace, it is
+              considered a Raster Foundry work and usage should follow the general
+              <a>terms of use</a>
+            </p>
+          </div>
+        </div>
+        <div class="horizontal-separator"></div>
+        <div class="model-detail-section">
+          <div class="model-detail-header">
+            <h3>Compatible Data Sources</h3>
+          </div>
+          <div>
+          </div>
+        </div>
+        <div class="model-detail-section">
+          <div class="model-detail-header">
+            <h3>Similar Models</h3>
+          </div>
+          <div class="list-group">
+            <rf-model-item
+                ng-repeat="modelData in $ctrl.similarQueryResult.results"
+                model-data="modelData"
+            ></rf-model-item>
+          </div>
+        </div>
+      </div>
+      <div class="column-3">
+        <div class="content">
+          <button class="btn btn-flat" style="width: 100%;">Save to My models</button>
+          <div class="panel">
+            <div class="panel-body centered">
+              <h5>Free to use</h5>
+              <button class="btn btn-primary">Execute Model</button>
+            </div>
+          </div>
+          <div class="reference-list">
+            <div class="reference">
+              Preview
+            </div>
+            <div class="reference">
+              Description
+            </div>
+            <div class="reference">
+              Upload Info
+            </div>
+            <div class="reference">
+              Requirements
+            </div>
+            <div class="reference">
+              Tags
+            </div>
+            <div class="reference">
+              Category
+            </div>
+            <div class="reference">
+              License
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<script type="text/ng-template" id="carousel.html">
```

Comment:
I think I'd prefer to have this as a separate template file if possible; it might get confusing later on to have two templates in one file.